tests/ga/test_extension.py

```diff
@@ -140,8 +141,6 @@ def test_cleanup_leaves_installed_extensions(self):
             exthandlers_handler.run()
             exthandlers_handler.report_ext_handlers_status()
 
-            self.assertEqual(no_of_exts, TestExtensionCleanup._count_packages(),
```

Comment:
The ZIP package is now deleted after expansion so all these tests do not need to check the count of packages